### DISCUSSION TITLE
Code from the documentation UI

### DISCUSSION THREAD

**Discussion by sudoFerraz** [wevm/wagmi#303] [Ideas]
Hello,

Currently I'm using a mix of react-moralis, wagmi and web3UIKit to display some good looking UI components.
I realized that the WAGMI documentation has been updated substantially, and I *really* liked the connect/connected components displayed on the documentation:

![image](https://user-images.githubusercontent.com/6979719/161078095-f973405d-7bc5-4222-9539-bca6a6f58643.png)

![image](https://user-images.githubusercontent.com/6979719/161078210-3f6f6338-da89-4ebd-b148-3cfb6868f9fe.png)

Any chance to add this components to the library?

If not, possible to share the code of them or any libraries being used + css?

BR;

**Comment by tmm** [comment]
The docs examples use [`degen`](https://github.com/mirror-xyz/degen).

**Reply by sudoFerraz** [reply]
Thank you @tmm 🙏
